pkg/sidecar_mounter/sidecar_mounter.go

```diff
@@ -427,15 +427,15 @@ func (m *Mounter) checkBucketAccessWithRetry(ctx context.Context, storageService
 		if ss == nil {
 			ss, err = m.StorageServiceManager.SetupStorageServiceForSidecar(ctx, tokenSource)
 			if err != nil {
-				klog.Errorf("Failed to setup storage service, got error %q, retrying...", err)
+				mc.ErrWriter.WriteMsg(fmt.Sprintf("%s: %v: %v %q, retrying...", util.SidecarBuckettAccessCheckErrorPrefix, util.StorageServiceErrorStr, storage.ParseErrCode(err), err))
 				return false, nil
 			}
 			klog.V(4).Infof("Created storage service %v", ss)
 		}
 
 		if bucketName != "_" {
 			if exist, err := ss.CheckBucketExists(ctx, &storage.ServiceBucket{Name: bucketName}); !exist {
-				klog.Errorf("Failed to get GCS bucket %q: %v", bucketName, err)
+				mc.ErrWriter.WriteMsg(fmt.Sprintf("%s: failed to get GCS bucket %s: %v %q", util.SidecarBuckettAccessCheckErrorPrefix, bucketName, storage.ParseErrCode(err), err))
```

Comment:
The formatting seems off here. Error is %q, and bucket name was changed from %q to %s. Should we keep it as %q to be consistent with how we log things in other places, or do you want it to be %s for string parsing?